fuel/src/main/kotlin/com/github/kittinunf/fuel/Fuel.kt

```diff
@@ -107,13 +124,15 @@ class Fuel {
 
         @JvmStatic
         fun request(convertible: RequestConvertible): Request = FuelManager.instance.request(convertible)
-
     }
-
 }
 
 @JvmOverloads
-fun String.httpGet(parameters: List<Pair<String, Any?>>? = null): Request = Fuel.get(this, parameters)
+fun String.httpGet(parameters: List<Pair<String, Any?>>? = null): Request = Fuel.get(this, parameters?.flatMap { pair ->
+    (pair.second as? Array<*>)?.let {
```

Comment:
We wanna try to get a higher abstraction? Maybe `iterable`? So that `List<*>` can be used as well

Reply:
I will update